packages/terra-functional-testing/lib/terra-dev-site/doc/terra-service/TerraService.doc.mdx

```diff
@@ -0,0 +1,64 @@
+# Terra Service
+
+The terra service integrates [axe accessibility](https://github.com/dequelabs/axe-core) testing into the test environment.
```

Comment:
Can we add info that this is a static analysis of the current dom structure & we are checking against these standards: 'wcag2a', 'wcag2aa', 'wcag21aa', 'section508'?

Reply:
Add more documentation to cover the standards we test against. I'm somewhat intentionally vague about how the command is run for now. I don't think anyone should be calling `browser.axe` directly, but it will eventually be wrapped by our chai assertions. 

https://github.com/cerner/terra-toolkit/pull/416/commits/2e4d70c646089a29c41346d30db7336ab1a5f41a